modules/siddhi-core/src/main/java/org/wso2/siddhi/core/util/transport/DynamicOptions.java

```diff
@@ -29,7 +29,7 @@ public DynamicOptions(Event event) {
         this.event = event;
     }
 
-    Event getEvent() {
+    public  Event getEvent() {
```

Comment:
When partitioning it should be possible to refer to the field value. How can we achieve this? Wouldn't it be okay if we make it public and make the return value final too?